README.md

```
@@ -177,100 +57,148 @@ export default [
+    A -->|mirrors rules| I[no-animal-violence-action]
+    A -->|mirrors rules| J[reviewdog-no-animal-violence]
+    A -->|mirrors rules| K[danger-plugin-no-animal-violence]
+```
 
----
+This repo is the single source of truth. Rule definitions live in three canonical files — `woke/.woke.yaml`, the `alex/` directory, and `vale/Speciesism/` — and are synchronized via `tools/check_consistency.py`. Downstream adapters implement the patterns in their tool's native format and are expected to track the core rules version. Any rule ID change in this repo requires downstream suppression comments to be updated; see [VERSIONS.md](VERSIONS.md) for the versioning policy.
 
-## Repository Structure
+**Repository structure:**
 
 ```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)